apps/backoffice-v2/src/pages/MerchantMonitoringUploadMultiple/hooks/useMerchantMonitoringUploadMultiplePageLogic/useMerchantMonitoringUploadMultiplePageLogic.tsx

```diff
@@ -0,0 +1,62 @@
+import { z } from 'zod';
+import { useNavigate } from 'react-router-dom';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+import csvContent from './batch-report-template.csv?raw';
+import { useLocale } from '@/common/hooks/useLocale/useLocale';
+import { CreateBatchBusinessReportSchema } from '@/pages/MerchantMonitoringUploadMultiple/create-batch-business-report-schema';
+import { useCreateBusinessReportBatchMutation } from '@/domains/business-reports/hooks/mutations/useCreateBusinessReportBatchMutation/useCreateBusinessReportBatchMutation';
+import { useCallback, useMemo } from 'react';
+
+export const useMerchantMonitoringUploadMultiplePageLogic = () => {
+  const form = useForm({
+    defaultValues: {
+      merchantSheet: undefined,
+    },
+  } as { defaultValues: { merchantSheet: File | undefined } });
+
+  const locale = useLocale();
+  const navigate = useNavigate();
+
+  const { mutate: mutateCreateBusinessReportBatch } = useCreateBusinessReportBatchMutation({
+    reportType: 'MERCHANT_REPORT_T1',
+    onSuccess: () => {
+      navigate(`/${locale}/merchant-monitoring`);
+    },
+  });
+
+  const onSubmit: SubmitHandler<z.output<typeof CreateBatchBusinessReportSchema>> = ({
+    merchantSheet,
+  }) => {
+    if (!merchantSheet) {
+      return;
+    }
+
+    mutateCreateBusinessReportBatch(merchantSheet);
+  };
+
+  const csvTemplateUrl = useMemo(() => {
+    const blob = new Blob([csvContent], { type: 'text/csv' });
+
+    return URL.createObjectURL(blob);
+  }, []);
```

Comment:
```suggestion
  }, [csvContent]);
```

Reply:
Nope! It is static, when I add it to the dependency list I get a warning :)